todo_app/templates/single_item.html

```diff
@@ -0,0 +1,18 @@
+{% extends "layout.html" %}
+{% block title %}To-Do App{% endblock %}
+
+{% block content %}
+  <div class="jumbotron">
+    <h1 class="display-4">To-Do App</h1>
+    <p class="lead">Just another to-do app.</p>
+  </div>
```

Comment:
If you want this on all the pages then you could try moving it into layout.html so you don't need to keep repeating it.